packages/terra-list/docs/List.md

```diff
@@ -0,0 +1,32 @@
+# Terra List
+
+The Terra List is a structural component to vertically arrange content within list/listitems.
+
+If a read-only list is desired, set the role prop for the list to 'none', as this will prevent accessibility concerns.
```

Comment:
Prevent accessibility concerns seem vague here. What are the main concerns associated to role="none"

Reply:
The default is 'listbox', but we don't want that role applied when the list is read-only. So passing null, or 'none' we not add a role to the list. You could however change it's role for something like 'group' for cases like menu.

Reply:
I can try to rephrase it.